nesta/core/orms/orm_utils.py

```diff
@@ -630,17 +630,23 @@ def insert_data(db_env, section, database, Base,
                                data=data,
                                low_memory=low_memory)
     objs, existing_objs, failed_objs = response
-    # Open a transaction
-    session = get_session(db_env, section, database, Base)
-    if merge_non_null:
-        session.execute(existing_objs)  # Drop existing objs if merging
```

Comment:
removed in favour of `db_session` context manager. The reason for this (other than it being cleaner), is explained below